### ISSUE TITLE
You are Breaking the GitHub TOS

### ISSUE THREAD

**Issue by HilkopterBob** [SuperMartijn642/ChunkLoaders#140]
Good Morning,

In Your mod "Chunk-Loaders" you are stating that it is 
_"protected under copyright law and may not be altered or reuploaded without direct permission"_ 

which is against the GitHub TOS which state that : 
_"If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content"_

[Link To TOS excertp](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#6-contributions-under-repository-license)

If you do not change your license to an open source License like GPL2 or GPL3 i will report your repository('s)
@SuperMartijn642

**Comment by SuperMartijn642** [comment]
What is the point of this random threatening message?

The mod is just ARR as I don't want people just taking it and reuploading the mod without putting in any effort. If there is something from Chunk Loaders that you would like to use, just ask. It is most likely fine with me.

> In Your mod "Chunk-Loaders" you are stating that it is
"protected under copyright law and may not be altered or reuploaded without direct permission"

Where would that be stated? As far as I know that is only in the description on CurseForge and Modrinth, not on GitHub.
The reason that message is on CurseForge and Modrinth is so I have something to point to when requesting my mod be removed from mod rehosting sites like 9minecraft.

> which is against the GitHub TOS which state that :
"If you set your pages and repositories to be viewed publicly, you grant each User of GitHub a nonexclusive, worldwide license to use, display, and perform Your Content"

Reading into the section you linked, I am not sure the statement on CurseForge and Modrinth goes against ToS. I looked it up and I found this [on the GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository) (under the 'Choosing the right license' section): 
"You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license."
